istio-1-3-1/istio-1-3-1/base/README.md

```diff
@@ -0,0 +1,7 @@
+# Manifest for Istio in Kubeflow
```

Comment:
Copied over `istio/istio` as `istio-1-3-1/istio-1-3-1` though not necessary.

Reply:
Removing `istio-1-3-1/istio-1-3-1` in hindsight, as this will cause config duplication and require devs to add changes to both folders.